[0m09:33:27.976364 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FA502948B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FA509E5910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FA52F32610>]}


============================== 09:33:27.981361 | 23a3f246-3fb6-455c-ab33-ce3dd6762a20 ==============================
[0m09:33:27.981361 [info ] [MainThread]: Running with dbt=1.7.0
[0m09:33:27.982362 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'C:\\Users\\test\\2023-12-bootcamp\\09-project-2\\project-2\\transform\\dw\\logs', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\test\\2023-12-bootcamp\\09-project-2\\project-2\\transform\\dw', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt run', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m09:33:27.985863 [error] [MainThread]: Encountered an error:
Runtime Error
  Could not find profile named 'northwind'
[0m09:33:27.988871 [debug] [MainThread]: Command `dbt run` failed at 09:33:27.988871 after 0.08 seconds
[0m09:33:27.990381 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FA502948B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FA52FC50A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FA52FC5160>]}
[0m09:33:27.992390 [debug] [MainThread]: Flushing usage events
[0m10:14:11.085385 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000186207644C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001861F79F1C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000186233FD190>]}


============================== 10:14:11.090896 | 3ded6a72-7840-446e-84b9-cfd13a8c906f ==============================
[0m10:14:11.090896 [info ] [MainThread]: Running with dbt=1.7.0
[0m10:14:11.091896 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': 'C:\\Users\\test\\2023-12-bootcamp\\09-project-2\\project-2\\transform\\dw', 'log_path': 'C:\\Users\\test\\2023-12-bootcamp\\09-project-2\\project-2\\transform\\dw\\logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt run', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m10:14:11.094896 [error] [MainThread]: Encountered an error:
Runtime Error
  Could not find profile named 'northwind'
[0m10:14:11.096897 [debug] [MainThread]: Command `dbt run` failed at 10:14:11.096897 after 0.08 seconds
[0m10:14:11.097928 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000186207644C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001862348F0D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001862348F190>]}
[0m10:14:11.098898 [debug] [MainThread]: Flushing usage events
[0m10:25:11.454326 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014E96A634C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014E996D49D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014E996D4760>]}


============================== 10:25:11.459508 | 2bbcc153-229b-4262-a72b-3ac81d6a7009 ==============================
[0m10:25:11.459508 [info ] [MainThread]: Running with dbt=1.7.0
[0m10:25:11.460540 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\test\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m10:25:11.461578 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\test\2023-12-bootcamp\09-project-2\project-2\transform\dw\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m10:25:11.463148 [debug] [MainThread]: Command `dbt run` failed at 10:25:11.463148 after 0.02 seconds
[0m10:25:11.463663 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014E96A634C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014E996D4760>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014E996D4A30>]}
[0m10:25:11.464175 [debug] [MainThread]: Flushing usage events
